PurchasesTests/Caching/DeviceCacheTests.swift

```diff
@@ -21,6 +21,10 @@ class DeviceCacheTests: XCTestCase {
                                        userDefaults: mockUserDefaults)
     }
 
+}
+
+extension DeviceCacheTests {
```

Comment:
This can be moved back in the original class.

Reply:
Done!